### PR TITLE
Fix #113224 golem.de

### LINKED ISSUE #113224: golem.de
### Issue URL (Ads)
 [https://www.golem.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.golem.de%2F)
### Comment
Username: @Morku
> There are 3 different placeholder which only appear on iOS (iPhone). On Desktop in mobile view, they are blocked (probably incompatible rules with iOS).
> 
> First is on homepage (Screenshot 1) `mob_afcbecoh`
> 
> Second on comments, URL: https://forum.golem.de/kommentare/sonstiges/m1-ultra-im-test-die-kroenung-des-apple-silicon/151671,list.html (Screenshot 2) `div[style]`
> 
> Third also in comments repeatedly (Screenshot 3) `mob_forum`
> 
> However, I tested around and found working solutions for iPhone and couldn't find any false blocking. Can you add/change them?
> 
> For better overview which placeholder, I disabled filtering on AdGuard Windows for the Screenshots.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.3
Browser: | Firefox
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard German

### PR DESCRIPTION
**Pull request by Morku**
First Fix Fix #113224

### CODE REVIEW

**Comment by BlazDT** [comment]
iOS uses optimized filters. These rules are already in German (non-optimized filter). :

golem.de##li[id^="mob_afcbecoh"]
forum.golem.de#?#.list-comments > li:has(> div[style])

Do you want to "optimize" them or should I in filter lists (as you already created pull request).
I see you found the 2nd rule already and changed them in your 3rd modification, was the rule itself not enough?

**Comment by Morku** [comment]
Missed the `mob_afcbecoh` in German filter since it not exist in Desktop view, only in `mob`ile.

Do I need to add `!+ NOT_OPTIMIZED` above the filter to "optimize"?
After I removed `.list-comments > ` for my personal filter test, the filter worked fine on my iPhone. Is it Optimize, too?

Feel free to adjust my pullrequest that I can learn from it.

**Comment by BlazDT** [comment]
Code is the same regarding the .list-comment for the selector.

Yes, add !+ NOT_OPTIMIZED in front of every single rule.

**Review by Morku — COMMENTED** [review]
changes

**Comment by Morku** [comment]
Please check again.

I still think `forum.golem.de##div[id^="mob_forum"]` and `golem.de##li[id^="mob_afcbecoh"]` should be either Mobile or German filters.

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -1155,6 +1155,7 @@ sibdom.ru##.fixed-position
 profile.ru###test_mob_ad
 ||hentaidude.com/wp-content/themes/awp/assets/mobile.js
 www-tellows-de.cdn.ampproject.org###singlecomments > li[style="padding:0px 5px 0px 5px;"]
+!+ NOT_OPTIMIZED
```
Comment (BlazDT) [inline MobileFilter/sections/specific_web.txt:1158]: Please move the rule and hint to the bottom where the other NOT_OPTMIZED rules are.

Move the two rules you mentioned in your comment to Mobile filter, name clearly indicates mobile, dunno why they are in normal filter.

**Review by Morku — COMMENTED** [review]
done

**Review by BlazDT — APPROVED** [review]

**Comment by BlazDT** [comment]
Thank you :)

**Pull request merged**